### PR TITLE
feat(client): enhance turn and time visibility on UI main player at the bottom

### PR DESCRIPTION
**Pull request by vkuprin**

### CODE REVIEW

File: GomokuClient/packages/gomoku-core/src/pages/JoinGame.tsx [hunk 1]
```diff
@@ -72,6 +71,36 @@ const JoinGame = ({ gameHistory }: JoinGameProps) => {
       });
       toaster.show("Rematch request sent");
     },
+    currentPlayer:
+      players.black?.playerId === jwtDecodedInfo?.userId ? "black" : "white",
```
Comment (Alequez97) [inline GomokuClient/packages/gomoku-core/src/pages/JoinGame.tsx:75]: Do we need to compare with anonymous generatedID as well, to handle this logic for anonymous case, or it will be for another PR?

**Review by Alequez97 — APPROVED** [review]

File: GomokuClient/packages/gomoku-core/src/pages/JoinGame.tsx (hunk 1 above)
Comment (vkuprin) [reply GomokuClient/packages/gomoku-core/src/pages/JoinGame.tsx]: Pushed fallback
